ota.c

```diff
@@ -754,13 +777,16 @@ int   ota_get_file_ex(char * repo, char * version, char * file, int sector, byte
                     if (header) {
                         //printf("%s\n-------- %d\n", recv_buf, ret);
                         //parse Content-Length: xxxx
-                        location=strstr(recv_buf,"Content-Length: ");
+                        location=strstr_lc(recv_buf,"content-length:");
                         strchr(location,'\r')[0]=0;
-                        location+=16; //flush Content-Length: //
+			if (location[15] == ' ') {
+                            location++;
+                        }
+                        location+=15; //flush Content-Length: //
```

Comment:
this string should also be preceded by a \n and numbers adjusted accordingly

Reply:
I don't include \n here because "content-length:" can not be confused with any other string. Thing that happens with "location:"

Reply:
I think that having \n is better for being more robust...